app/main.py

```diff
@@ -1,7 +1,37 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import os
+import requests
+
+
+API_URL = "http://api.weatherapi.com/v1/current.json"
+CITY = "Paris"
```

Comment:
Code Style Issue: The names of your constants should be more descriptive. 'API_URL' and 'CITY' are too generic. Consider renaming them to something like 'WEATHER_API_URL' and 'DEFAULT_CITY' respectively.